MovingBlocks/Terasology
# Update select game screen

### Contains

**Note**: work still in progress. I just want to get feedback from mentors as soon as they can.

This is initial PR for first week of GSoC which covers next features:
* updates of view SelectGameScreen
* present short description of selected saved game
* load preview image or show default image
* capture preview image, before user leave game

### How to test

1. Open SelectGameScreen. 
2. Load some saved game.
3. Exit to Main screen.
4. Open SelecteGameScreen again and check that for the saved game now you have a preview.

Also you might check that GameManifest of the saved game has the same data (worldGenerator and list of modules). 

### Outstanding before merging

If anything. You can use neat checkboxes! Feel free to delete if not needed

- [x] Improve UI of the SelectGameScreen
- [x] Change default preview image
